### ISSUE TITLE
Overlay doesn't show in EVE Online

### ISSUE THREAD

**Issue by mkrautz** [mumble-voip/mumble#3117]
Via ksignorini http://forums.mumble.info/viewtopic.php?f=4&t=3318:
> When running Mumble 1.2.19 or 1.3.0~2318~g8f65051~snapshot (latest snapshot as of this writing) the Overlay will not show up in the EVE Online game client on the Mac.
> 
> It does, however, show the Overlay in the EVE Launcher window.
> 
> Does this have something to do with how EVE uses WINE only now? If so, any solutions to make it show up > in the game and not the Launcher window?
> 
> Thank you.

**Comment by bahrmichael** [comment]
I'd like to help with this issue. Can you point me in the right direction for code that handles the overlay?

**Comment by davidebeatrici** [comment]
Main code: https://github.com/mumble-voip/mumble/blob/master/src/mumble/Overlay.cpp

macOS code:
https://github.com/mumble-voip/mumble/blob/master/src/mumble/Overlay_macx.mm
https://github.com/mumble-voip/mumble/blob/master/macx/osax/osax.m
https://github.com/mumble-voip/mumble/blob/master/overlay_gl/init_mac.c

**Comment by bahrmichael** [comment]
I'm on hold until master is green again. Wasn't able to get the build running locally and therefore rely on travis.

**Comment by davidebeatrici** [comment]
Build fixed, sorry for the delay.

**Comment by fnordpig** [comment]
Did anything ever get merged on this @bahrmichael ? If not can I pick it up from you?

**Comment by Krzmbrzl** [comment]
Given the long period of inactivity here I strongly suspect that no (considerable) progress has been made. I might be wrong though...

**Comment by bahrmichael** [comment]
Yeah, no progress on my side.

**Comment by fnordpig** [comment]
OK I'll start to take a look - not sure how that works with this project haven't read the contribute docs etc.  If I fall off the horizon no harm no foul but just putting my hand up to try.

**Comment by Krzmbrzl** [comment]
> not sure how that works with this project haven't read the contribute docs

We don't have strict contribution guidelines. If you create a (draft) PR early on though we can try to assist you in the process as good as possible.
More or less the only formal guideline that we have is our commit guideline to be found at the repo's root. That is only important once the PR is finished though. :shrug: 
And of course: Never hesitate to ask. We'll do our best to provide usable answers :)

**Comment by KaiNeon** [comment]
@davidebeatrici 
My issue was closed very quickly, attached here, but even the operating system did not match before your change.
Maybe I can provide more information or some logs?
Also different is that the overlay is not shown in the launcher too, in my case.

**Comment by KaiNeon** [comment]
The issue was temporarily resolved by disabling the "use DirectX 12" and "use hardware rendering" options in the game launcher.

**Comment by mike0891** [comment]
@davidebeatrici These issues are unrelated. 

Mumble overlay not working, is because of Eve Online moved to DX 12 this week. It has nothing to do with an issue from half a decade ago.

**Comment by davidebeatrici** [comment]
Okay, sorry. Let's reuse this issue anyway, as it was pretty much abandoned.

**Comment by Krzmbrzl** [comment]
Since this is due to missing DX12 support, this is a duplicate of #5760
